src/domain/discounts/details/conditions/edit-condition/condition-table-actions.tsx

```diff
@@ -0,0 +1,69 @@
+import clsx from "clsx"
+import React from "react"
+import { useContext } from "react"
+import Button from "../../../../../components/fundamentals/button"
+import PlusIcon from "../../../../../components/fundamentals/icons/plus-icon"
+import { LayeredModalContext } from "../../../../../components/molecules/modal/layered-modal"
+import { useAddConditionsModalScreen } from "./add-conditions-screens"
+import { useEditConditionContext } from "./edit-condition-provider"
+
+const ExistingConditionTableActions = ({
+  numberOfSelectedRows,
+  onDeselect,
+  onRemove,
+}: any) => {
+  const { condition } = useEditConditionContext()
+
+  const addConditionsModalScreen = useAddConditionsModalScreen(condition)
+
+  const showAddConditions = !!numberOfSelectedRows
+
+  const classes = {
```

Comment:
Yes, the approach is used elsewhere, but I'd like to keep it out of this PR in fear of overengineering something that is already quite simple to work with

Reply:
it would just be a simple util that does exactly that depending on the input condition. The idea is to have a consistent transition everywhere. We can see that later